tests/hf/test_dipole.py

```diff
@@ -0,0 +1,361 @@
+# Copyright 2018-2022 Xanadu Quantum Technologies Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+Unit tests for functions needed for computing the dipole.
+"""
+import autograd
+import pennylane as qml
+import pytest
+from pennylane import Identity, PauliX, PauliY, PauliZ
+from pennylane import numpy as np
+from pennylane.hf.dipole import (
+    dipole_integrals,
+    dipole_moment,
+    fermionic_dipole,
+    fermionic_one,
+    qubit_operator,
+)
+from pennylane.hf.molecule import Molecule
+
+
+@pytest.mark.parametrize(
+    ("symbols", "geometry", "charge", "core", "active", "core_ref", "int_ref"),
```

Comment:
Since ``charge`` is not changed in the tests, no need to parametrize here, right? You can also leave it and include water for a given active space?

Reply:
We actually have H3+ in the test.